lib/ansible/modules/cloud/amazon/sqs_queue.py

```diff
@@ -139,178 +217,184 @@
 '''
 
 import json
-import traceback
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import camel_dict_to_snake_dict, compare_aws_tags, snake_dict_to_camel_dict
 
 try:
-    import boto.sqs
-    from boto.exception import BotoServerError, NoAuthHandlerFound
-    HAS_BOTO = True
-
+    from botocore.exceptions import BotoCoreError, ClientError, ParamValidationError
 except ImportError:
-    HAS_BOTO = False
-
-from ansible.module_utils.basic import AnsibleModule
-from ansible.module_utils.ec2 import AnsibleAWSError, connect_to_aws, ec2_argument_spec, get_aws_connection_info
-
-
-def create_or_update_sqs_queue(connection, module):
-    queue_name = module.params.get('name')
+    pass  # handled by AnsibleAWSModule
+
+ARGUMENT_SPEC = dict(
```

Comment:
Could you change this to define the arg_spec in main() rather than via a global var?